src/mots/cli.py

```diff
@@ -52,7 +52,7 @@ def validate(args: argparse.Namespace) -> None:
     file_config = config.FileConfig(Path(args.path))
     file_config.load()
     errors = config.validate(file_config.config, file_config.repo_path)
-    for error in errors:
+    for error in errors or []:
```

Comment:
Nit: I'd put this on the above line so `errors` is always a `list` instead of `Optional[list]`:
```python
errors = config.validate(file_config.config, file_config.repo_path) or []
```